oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOpaqueTokenIntrospectConditional.java

```diff
@@ -24,12 +24,12 @@
         super(ConfigurationPhase.REGISTER_BEAN);
     }
 
-    @ConditionalOnProperty(name="okta.oauth2.client-id")
+    @ConditionalOnProperty(name="spring.security.oauth2.resourceserver.opaque-token.client-id")
     static class ClientIdCondition { }
 
-    @ConditionalOnProperty(name="okta.oauth2.client-secret")
+    @ConditionalOnProperty(name="spring.security.oauth2.resourceserver.opaque-token.client-secret")
     static class ClientSecretCondition { }
 
-    @ConditionalOnProperty(name="okta.oauth2.issuer")
-    static class IssuerCondition { }
+    @ConditionalOnProperty(name="spring.security.oauth2.resourceserver.opaque-token.introspection-uri")
+    static class IntrospectionUriCondition { }
```

Comment:
Great name update! But is this condition used anywhere?

Reply:
This is the third of thee opaque-token conditions. For correct initialization of OpaqueTokenIntrospector, they all have to be provided. So I think we need it.